README.md

```diff
@@ -35,30 +76,45 @@ The following **optional** CMake variables can be added:
 
 Then build the project in the build directory:
 
-`cmake --build . --target install`
+```bash
+$> cmake --build . --target install
+```
 
 ## Run
 To run DynaFlow launcher, use the script dynaflow-launcher.sh provided with the installation. This script will set the required environment variables.
 
 Runtime options of binary are given by doing:
 
-`dynaflow-launcher.sh --help`
+```bash
+$> ./dynaflow-launcher.sh --help
+```
+from the install directlory
 
-Only network file and configuration are required
+Only network file and configuration are required.
+
+You can also use the script with the "launch" or "launch-dir" option.
 
 ## Testing
 DynaFlow Launcher testing relies on cmake tests. To launch units tests, launch from *build* directory:
 
-`ctest -j<N> --output-on-failure`
+```bash
+$> ctest -j<N> --output-on-failure
```

Comment:
Since this is available in the script let's propose always the myEnvDFL.sh option instead of cmake commands

Reply:
OK